tutorials/applications/QuantumEvolutionKernel_Tutorial.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
```

Comment:
<p>The link is not working, I think because you are linking to <code>#application</code>  instead of <code>#Application-on-a-benchmark-dataset</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>